modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/components/page-accessibility-tool/components/_Violations.scss

```diff
@@ -0,0 +1,26 @@
+@import 'atlas-variables';
+
+.page-accessibility-tool__sidebar {
+	width: 320px;
+
+	&--violations-panel-header-title {
+		border-bottom: 0.0625rem solid #f2f2f2;
+		border-color: #e7e7ed;
+		margin-left: 1rem;
+		margin-right: 1rem;
+		margin-top: 0;
+		padding: 0.5rem 0;
+	}
+
+	&--violations-panel-header-description {
+		padding: 1rem;
+	}
+
+	&--rule-list-item {
+		padding-left: inherit !important;
+	}
+
+	.list-group-item {
+		border: 0.0625rem solid transparent;
```

Comment:
If you just want to leave the transparent border to rewrite, use `border-color: transparent;`, it is safer when there is a change in the main class.